### ISSUE TITLE
Install failure

### ISSUE THREAD

**Issue by Doinng** [coreybutler/nvm-windows#782]
PS C:\Users\Kite> nvm install 16.14.2
Downloading node.js version 16.14.2 (64-bit)...
Complete
Creating C:\Users\Kite\AppData\Roaming\nvm\temp

Downloading npm version 8.5.0... Error while downloading https://github.com/npm/cli/archive/v8.5.0.zip - Get https://github.com/npm/cli/archive/v8.5.0.zip: net/http: TLS handshake timeout
panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xc0000005 code=0x0 addr=0x20 pc=0x5c6eb0]

goroutine 1 [running]:
_/C_/Users/corey/Documents/workspace/nvm-windows/src/nvm/web.Download(0x11374000, 0x2d, 0x1123a180, 0x35, 0x113eb390, 0x5, 0x0)
        C:/Users/corey/Documents/workspace/nvm-windows/src/nvm/web/web.go:68 +0x240
_/C_/Users/corey/Documents/workspace/nvm-windows/src/nvm/web.GetNpm(0x11238571, 0x21, 0x113eb390, 0x5, 0x3b)
        C:/Users/corey/Documents/workspace/nvm-windows/src/nvm/web/web.go:163 +0x21b
main.install(0x1123e0a8, 0x7, 0x660283, 0x2)
        C:/Users/corey/Documents/workspace/nvm-windows/src/nvm.go:268 +0x626
main.main()
        C:/Users/corey/Documents/workspace/nvm-windows/src/nvm.go:78 +0x8cd
PS C:\Users\Kite> nvm install 16.14.2
Version 16.14.2 is already installed.
PS C:\Users\Kite> nvm
Running version 1.1.7.

Usage:

  nvm arch                     : Show if node is running in 32 or 64 bit mode.
  nvm install <version> [arch] : The version can be a node.js version or "latest" for the latest stable version.
                                 Optionally specify whether to install the 32 or 64 bit version (defaults to system arch).
                                 Set [arch] to "all" to install 32 AND 64 bit versions.
                                 Add --insecure to the end of this command to bypass SSL validation of the remote download server.
  nvm list [available]         : List the node.js installations. Type "available" at the end to see what can be installed. Aliased as ls.
  nvm on                       : Enable node.js version management.
  nvm off                      : Disable node.js version management.
  nvm proxy [url]              : Set a proxy to use for downloads. Leave [url] blank to see the current proxy.
                                 Set [url] to "none" to remove the proxy.
  nvm node_mirror [url]        : Set the node mirror. Defaults to https://nodejs.org/dist/. Leave [url] blank to use default url.
  nvm npm_mirror [url]         : Set the npm mirror. Defaults to https://github.com/npm/cli/archive/. Leave [url] blank to default url.
  nvm uninstall <version>      : The version must be a specific version.
  nvm use [version] [arch]     : Switch to use the specified version. Optionally specify 32/64bit architecture.
                                 nvm use <arch> will continue using the selected version, but switch to 32/64 bit mode.
  nvm root [path]              : Set the directory where nvm should store different versions of node.js.
                                 If <path> is not set, the current root will be displayed.
  nvm version                  : Displays the current running version of nvm for Windows. Aliased as v.

PS C:\Users\Kite> nvm uninstall 16.24.0
node v16.24.0 is not installed. Type "nvm list" to see what is installed.

**Comment by coreybutler** [comment]
In the future, please follow the issue template (it's provided for a reason).

The problem is `net/http: TLS handshake timeout`. This means your internet connection is failing to communicate with Github (`https://github.com/npm/cli/archive/v8.5.0.zip`). This often happens when using a proxy.

You can bypass some of the TLS requirements using the `--insecure` flag, as defined in the help message:

<img width="832" alt="image" src="https://user-images.githubusercontent.com/770982/161799232-4f188d3e-592b-4bac-8356-dc4648245d4f.png">

In other words, run `nvm install --insecure 16.14.2`.
